include/aspect/global.h

```diff
@@ -190,6 +190,21 @@ namespace aspect
   extern const bool output_parallel_statistics;
 
 
+  namespace citation
```

Comment:
maybe Capitalize? I think we only used lower case namespace names if it's a proper name (dealii, aspect).